src/Files.App/Views/Properties/MainPropertiesPage.xaml

```diff
@@ -145,7 +146,7 @@
 					</Grid.RowDefinitions>
 
 					<!--  Frame  -->
-					<Frame x:Name="MainContentFrame" IsNavigationStackEnabled="False" />
```

Comment:
Why was this removed? I'm pretty sure this was intentional.

Reply:
if IsNavigationStackEnabled is enabled, frame cannot go back by clicking Back Button placed on the top of the window.